php/Blocks/Loader.php

```diff
@@ -610,4 +582,30 @@ public function add_field( $block_name, $field_config ) {
 
 		$this->blocks[ "genesis-custom-blocks/{$block_name}" ]['fields'][ $field_config['name'] ] = $field_config;
 	}
+
+	/**
+	 * Adds 'POST' to the allowed REST methods for GCB blocks.
+	 *
+	 * The <ServerSideRender> uses the httpMethod of 'POST' to handle a larger attributes object.
+	 * That is already added in WP 5.6+, so no need to add it there.
+	 *
+	 * @todo: Delete when this plugin's 'Requires at least' is bumped to 5.6.
```

Comment:
Good idea to add this reminder. Will be nice when we can bump the minimum version to 5.6.